Public/Src/Utilities/Utilities/RelativePath.cs

```diff
@@ -169,14 +169,17 @@ public static ParseResult TryCreate<T>(StringTable table, T relativePath, out Re
                                 || (relativePath[index + 2] == '/'))
                             {
                                 // component is a sole .. so try to go up
-                                if (components.Count == 0)
+                                if (components.Count == 0 && !isAbsolute)

```

Comment:
>isAbsolute [](start = 62, length = 10)

I'm thinking it would be good to rename the isAbsolute parameter to something that more closely describes the behavioral difference of the validation provided by TryCreate. What do you think about "failDotDotOutOfScrope"?

Reply:
I think fromAbsolutePath is more accurate here, since if the components.Count == 0 && the function was not called form AbsolutePath then we should return the error.

Reply:
Changing it to "fromAbsolutePath" in next commit

Reply:
Discussed a tweak offline